rest_framework_json_api/renderers.py

```diff
@@ -396,7 +396,7 @@ def build_json_resource_obj(cls, fields, resource, resource_instance, resource_n
             resource_data.append(('links', {'self': resource[api_settings.URL_FIELD_NAME]}))
         return OrderedDict(resource_data)
 
-    def render_relationship_view(self, data, accepted_media_type=None, renderer_context=None):
+    def render_relationship_view(self, data, accepted_media_type=None, renderer_context={}):
```

Comment:
Using a mutable default parameter is generally a bad idea in python, due to:

> Python's default arguments are evaluated once when the function is defined, not each time the function is called (like it is in say, Ruby). This means that if you use a mutable default argument and mutate it, you will and have mutated that object for all future calls to the function as well.

[more details](https://www.google.com.ar/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&cad=rja&uact=8&ved=0ahUKEwiI7uyWk4HSAhUGjJAKHRWMAGIQFggcMAE&url=http%3A%2F%2Fdocs.python-guide.org%2Fen%2Flatest%2Fwriting%2Fgotchas%2F&usg=AFQjCNF849mrqlMC96-oZkr80MSgOb_R4w)